newspaper/images.py

```diff
@@ -167,6 +171,52 @@ def fetch_image_dimension(url, useragent, referer=None, retries=1):
     return fetch_url(url, useragent, referer, retries, dimension=True)
 
 
+def valid_image_url(url):
```

Comment:
This is great work! But it's also used as a core function, maybe we can add a few simple unit tests for this function on say 10 good image URLs and 10 bad image urls just so we have sanity knowing it works?

Reply:
will do

Reply:
added, see below